README.md

```diff
@@ -38,7 +38,7 @@ pip install --pre torch torchvision torchtext torchaudio -f https://download.pyt
 
 Install other necessary libraries:
 ```
-pip install pyyaml
+pip install pyyaml pytest
```

Comment:
If `distro` is also missing, we should add it here too.

Reply:
Actually I noticed that pytest, distro, etc. are included in requirements.txt. Not sure how come it wasn't installed as part of `python install.py`. I will double check

Reply:
seems like its my fault and the initial install.py didnt succeed for me. Now i see the packages. I'll remove the package update and only check in the typo change

Reply:
Sounds good to me, thanks!